uavcan/driver/python_can.py

```diff
@@ -0,0 +1,157 @@
+#
+# Copyright (C) 2019 UAVCAN Development Team  <uavcan.org>
+#
+# This software is distributed under the terms of the MIT License.
+#
+# Author: Jos Vos <jos.vos@demcon.nl>
+#
+
+from __future__ import division, absolute_import, print_function, unicode_literals
+import os
+import errno
+import time
+import threading
+from logging import getLogger
+
+from .common import DriverError, TxQueueFullError, CANFrame, AbstractDriver
+from .timestamp_estimator import TimestampEstimator
+
+try:
+    import queue
+except ImportError:
+    # noinspection PyPep8Naming,PyUnresolvedReferences
+    import Queue as queue
+
+logger = getLogger(__name__)
+
+try:
+    import can
+except ImportError:
+    logger.error("no module 'can' available, please install python-can")
+
+
+class PythonCan(AbstractDriver):
+    TX_QUEUE_SIZE = 1000
+
+    def __init__(self, interface, **_extras):
+        super(PythonCan, self).__init__()
+
+        # get bus from default config
+        try:
+            self._bus = can.Bus()
+            self._bus.state = can.bus.BusState.ACTIVE
+        except Exception as ex:
+            logger.error("Got an exception", exc_info=True)
+
+        self._writer_thread_should_stop = False
+        self._write_queue = queue.Queue(self.TX_QUEUE_SIZE)

```

Comment:
Would it be okay to do without this queue?

Reply:
As I understand, Python-CAN may be blocking, so I wouldn't remove the queue to keep the semantics of our various drivers compatible.